docs/content/tutorial-cmake.md

```diff
@@ -0,0 +1,146 @@
+CMake Tutorial {#tut_cmake}
+====
+\brief Generating LCM type bindings with CMake
+
+This tutorial will walk you through a sample `CMakeLists.txt` that will
+generate bindings for your LCM message types.
+
+First, of course, you will want to find LCM itself, and include it's "use file"
+(a "use file" is a CMake script provided with some packages that adds utility
+functions for using that package). This can be done in your root
+`CMakeLists.txt`. If you will be generating Python and/or Java bindings, now
+is a good time to find those also.
+
+    find_package(lcm REQUIRED)
+    include(${LCM_USE_FILE})
+
+    find_package(PythonInterp)
+    find_package(Java)
+
+    if(JAVA_FOUND)
+      include(UseJava)
+    endif()
+
+Add `REQUIRED` to the `find_package` calls if you want to enforce that Python
+and/or Java are available. Otherwise, you can generate bindings for these
+languages conditionally. (The rest of the logic we show, including the next
+snippet, will go in the `CMakeLists.txt` that's located alongside your LCM
+types.)
+
+    if(PYTHONINTERP_FOUND)
+      set(python_args PYTHON_SOURCES python_install_sources)
+    endif()
+    if(JAVA_FOUND)
+      set(java_args JAVA_SOURCES java_sources)
+    endif()
+
+(The main reason to find Python is to obtain its version so that the generated
+Python files can be installed into the correct directory. If you determine this
+directory by some other means, you can skip finding Python and just generate
+the Python bindings unconditionally.)
+
+Now, you'll want to generate the bindings.
+
+    lcm_wrap_types(
+      C_SOURCES sources
+      C_HEADERS c_install_headers
```

Comment:
I've used `_install_` in those variables that will receive lists of files that will be installed, as opposed to just used to build a library (e.g. `c_sources`). It's certainly not necessary, but it's a potentially useful indicator. Do you have a strong feeling either way? (I can mention this reasoning in the changes I'll be making per @tkoolen's requests.)